mix.exs

```diff
@@ -37,7 +37,7 @@ defmodule Membrane.FFmpeg.SWScale.Mixfile do
       {:membrane_core, "~> 0.12.8"},
       {:membrane_raw_video_format, "~> 0.3.0"},
       {:membrane_common_c, "~> 0.15.0"},
-      {:unifex, "~> 1.0"},
+      {:bundlex, "~> 1.2"},
```

Comment:
```suggestion
      {:bundlex, "~> 1.2.0"},
```

Reply:
Why?

Reply:
my mistake, I got too used to working with `0.x.y` versions